### DISCUSSION TITLE
Welcome to Citizen-Web3 Discussions!

### DISCUSSION THREAD

**Discussion by serejandmyself** [citizenweb3/citizenweb3#59] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by serejandmyself** [comment]
Hey! Feel free to drop any comments here about what we do. Our docs. Our issues, etc

**Reply by zhangzuohong** [reply]
感觉非常不错！
